packages/fcl/src/wallet-provider-spec/draft-v3.md

```diff
@@ -0,0 +1,548 @@
+## Status
+
+- **Last Updated:** August 3rd 2021
+- **Stable:** Yes
+- **Risk of Breaking Change:** Medium
+- **Compatibility:** `>= @onflow/fcl@0.0.77`
+
+# Overview
+
+Flow Client Library (FCL) approaches the idea of blockchain wallets on Flow in a different way than how wallets may be supported on other blockchains. For example, with FCL, a wallet is not necessarily restricted to being a browser extention or even a native application on a users device. Wallets on Flow can take the shape of being many different types applications. Since wallet applications can take on many forms, we needed to create a way for these varying applications to be able to communicate and work together.
+
```

Comment:
For example, with FCL, a wallet is not necessarily LIMITED to being a browser extention or even a native application on a users device. FCL offers wallet developers the flexibility and freedom to build many different types of applications.
Since wallet applications can take on many forms, we needed to create a way for these varying applications to be able to communicate and work together.